basics/keeping-up-to-date/update/update-from-the-back-office.md

```diff
@@ -0,0 +1,244 @@
+  corresponds to the most recent version of PrestaShop compatible with the PHP version of your server.
+- The “local” update, which displays customized updates detected in your /your-admin-directory/autoupgrade/download
+  folder on your server.
+
+These views are thus adapted to the version of PrestaShop used in your store, to the PHP version of your server, and
+therefore reflect possible updates for your store.
+
+![Version choice page](../img/update-assistant-version-choice-page.png)
+
+If you're using the local channel, choose the .zip archive and associated .xml file to be guided through the update
+process. Be sure to choose a .zip and .xml file with the corresponding PrestaShop version (otherwise you'll get an error
+message).
+
+![Version choice local channel](../img/update-assistant-version-choice-local-channel.png)
+
+In all cases, when selecting an update channel, a check of the technical prerequisites is performed before proceeding to
+the next step:
+
+- Error messages that block access to the next step are displayed with a red cross at the beginning of the line.
+- Warning messages, which allow access to the next step, are displayed with an orange warning at the beginning of the
+  line.
+
+Please consider these points before proceeding to the next step, as they may affect the stability of your store.
+
+![Check requirements](../img/update-assistant-check-requirements.png)
+
+Once the update channel has been selected and the technical prerequisites checked (non-blocking), you can click on the
+“Next” button to access the next step, “Update options”.
+
+### Update options
+
+During this step you can configure the options to be applied to your update.
+
+![Update options page](../img/update-assistant-update-options-page.png)
+
+Here are the available options:
+
+- **Deactivate non-native modules:** this option allows you to deactivate all modules installed after the creation of
+  your store and therefore qualified as “non-native”. These modules may cause instabilities with the target version of
+  the PrestaShop update, which is why we recommend that you disable them (set to “yes” by default).
+- **Regenerate email templates:** this option allows email templates to be regenerated during the update process (set to
+  “yes” by default). Be careful, if you have customized email templates, activating this option may cause you to lose
+  your modifications.
+- **Disable all overrides**: this option allows you to directly disable all class and controller overrides (set to “no”
+  by default). Enabling this option helps prevent conflicts arising during and after the update process.
+
+Once the update options have been configured, you can click on the “Next” button to access the next step, “Backup”.
+
+### Backup
+
+During this step, you can choose whether or not to make a backup of your store with the Update Assistant module. Backing
```

Comment:
I think this paragraph could mention that manual backup is also strongly recommended.